etc/firejail-default

```diff
@@ -95,15 +99,22 @@ owner /run/firejail/mnt/oroot/{run,dev}/shm/** rmwk,
 /proc/@{PID}/statm r,
 /proc/@{PID}/status r,
 /proc/@{PID}/task/@{PID}/stat r,
+/proc/@{PID}/task/@{PID}/status r,
 /proc/@{PID}/maps r,
+/proc/@{PID}/mem r,
 /proc/@{PID}/mounts r,
 /proc/@{PID}/mountinfo r,
+owner /proc/@{PID}/oom_adj w,
 /proc/@{PID}/oom_score_adj r,
+owner /proc/@{PID}/oom_score_adj w,
 /proc/@{PID}/auxv r,
 /proc/@{PID}/net/dev r,
 /proc/@{PID}/loginuid r,
 /proc/@{PID}/environ r,
 
+# Needed for chromium
+ptrace (trace tracedby),
```

Comment:
afaik only the chromium crash handler needs this.

Reply:
Should we allow it or not? Maybe leave it commented out it by default same as dbus?